parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -442,4 +442,10 @@ datasource:
         apply_network_config: false
 EOF
 }
+
+addUdevDriverRule() {
+    cat << EOF > tmp
+SUBSYSTEM=="net", SUBSYSTEMS=="vmbus", DRIVERS=="hv_netvsc", ENV{ID_NET_DRIVER}="hv_netvsc"
```

Comment:
```suggestion
SUBSYSTEM=="net", SUBSYSTEMS=="vmbus", DRIVERS=="hv_netvsc", ENV{ID_NET_DRIVER}="$driver"
```

This is a little cleaner by referencing the driver - it pulls from the parent instead of risking a typo.